upgrade/1.0.1/scripts/upgrade/add_pod_priority.sh

```diff
@@ -91,6 +91,15 @@ function wait_for_running_pods() {
     current_running_num=$(kubectl get po -n $ns | grep $etcd_cluster | grep Running | wc -l)
     if [[ "$desired_size" -eq "$current_running_num" ]]; then
       echo "Found $desired_size running pods in $etcd_cluster etcd cluster"
+      while true; do
```

Comment:
This could end up in an infinite loop. Do you want to set a loop count limit and exit the loop with failure to prevent it from happening?

Reply:
good question -- i do like that it'll just spin -- that will force installers to interrupt and address this issue before moving on (in the same way the outer loop could also spin).

Reply:
That sounds good. I'll approve.